src/models/Project/index.js

```diff
@@ -38,6 +47,11 @@ class Project extends Model {
   description: string
   owner: User
   locales: Array<Locale>
+  translationKeys: Array<TranslationKey>
+
+  hasOwnerId(ownerId: string): boolean {
+    return this.ownerId === ownerId
+  }
```

Comment:
Now I’m actually thinking it’s not so useful since it only does that… 😄What do you think?